tfx/dsl/placeholder/placeholder.py

```diff
@@ -531,6 +531,28 @@ def __init__(self):
     super().__init__(placeholder_pb2.Placeholder.Type.EXEC_INVOCATION)
 
 
+class EnvironmentVariablePlaceholder(Placeholder):
+  """Environment Variable Placeholder helps access EnvironmentVariable proto.
+
+  Prefer to use environment_variable(...) to create Environment Variable
+  placeholder.
+  """
+
+  def __init__(self, key):
+    super().__init__(placeholder_pb2.Placeholder.Type.ENVIRONMENT_VARIABLE, key)
+
+
+class StringValuePlaceholder(Placeholder):
```

Comment:
If the goal is to concatenate string with EnvironmentVariablePlaceholder, there is no need to add StringValuePlaceholder. It should already be supported because [Placeholder](https://source.corp.google.com/piper///depot/google3/third_party/py/tfx/dsl/placeholder/placeholder.py;bpv=1;bpt=1;l=332?gsn=Placeholder&gs=kythe%3A%2F%2Fgoogle3%3Flang%3Dpython%3Fpath%3Dthird_party%2Fpy%2Ftfx%2Fdsl%2Fplaceholder%2Fplaceholder.py%23module.Placeholder) base class overrides add and radd.